GeosCore/carbon_gases_mod.F90

```diff
@@ -1241,21 +1035,16 @@ SUBROUTINE Init_Carbon_Gases( Input_Opt,  State_Chm, State_Diag,             &
 
     !========================================================================
     ! Define GEOS-Chem species indices
-    ! NOTE: Some of these are for tagged species, which are deactivated
-    ! by default.  Interested users can add tagged species if they wish.
     !========================================================================
-    id_CH4     = Ind_( 'CH4'     )
-    id_CO      = Ind_( 'CO'      )
-    id_COacet  = Ind_( 'COacet'  )
-    id_COch3oh = Ind_( 'COch3oh' )
-    id_COch4   = Ind_( 'COch4'   )
-    id_COisop  = Ind_( 'COisop'  )
-    id_COmono  = Ind_( 'COmono'  )
-    id_COnmvoc = Ind_( 'COnmvoc' )
-    id_CO2     = Ind_( 'CO2'     )
-    id_CO2ch   = Ind_( 'CO2ch'   )
-    id_OCS     = Ind_( 'OCS'     )
-    id_OH      = Ind_( 'FixedOH' )
+    id_CH4        = Ind_( 'CH4'        )
+    id_CH4_adv    = Ind_( 'CH4',   'A' )
+    id_CO         = Ind_( 'CO'         )
+    id_CO_adv     = Ind_( 'CO',    'A' )
+    id_CO2        = Ind_( 'CO2'        )
+    id_CO2_adv    = Ind_( 'CO2',   'A' )
+    id_OCS        = Ind_( 'OCS'        )
+    id_OCS_adv    = Ind_( 'OCS',   'A' )
+    id_OH         = Ind_( 'FixedOH'    )
```

Comment:
It looks like tagged CO is removed. That should be in the changelog if anyone in the past has used it.

Reply:
@lizziel @msulprizio: The tagged code never really worked as it was always blocked out.  We can add it to the changelog though.

Reply:
Yeah, something like "Removed unused tagged CO code" would be good to document in case anyone later wonders what happened to it.

Reply:
I added the removal of tagged CO and CH4 code to CHANGELOG.md in https://github.com/geoschem/geos-chem/pull/1992/commits/e837aba3ec9d112b4ab4cc501707e621e43db042.